OneCommunityGlobal/HighestGoodNetworkApp
# Shihao - Team member sort function

# Description
![Screenshot 2023-08-18 at 13 52 18](https://github.com/OneCommunityGlobal/HGNRest/assets/62367016/be665c8e-aa87-4aaf-b430-1c5db811661f)

## Related PRS:
This frontend PR is related to the [#474](https://github.com/OneCommunityGlobal/HGNRest/pull/474) backend PR.

## Main changes explained:
- Sort members by alphabet by default
- Create a column `Date Added`
- Create a way to sort members by date

## How to test:
1. check into current branch
2. do `npm install` and `npm start` to run this PR locally
3. log as admin/owner user
4. go to dashboard→ Teams→ Members
5. check whether the member is sorted by alphabet
6. click the 'Date Added' to check if the user is sorted by both date and alphabet

## Screenshots or videos of changes:

https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/62367016/72c9f985-a44b-419e-9d79-d474dbb73c44
